apps/backoffice-v2/src/domains/business-reports/components/Transactions/Transactions.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react';
+import { ArrowRight } from 'lucide-react';
+
+import { PremiumSvg } from '@/common/components/atoms/icons';
+import { Link } from 'react-router-dom';
+
+export const Transactions: FunctionComponent = () => (
+  <div className={`flex flex-row`}>
+    <img
+      alt={`Transaction Analysis`}
+      src={'/images/TransactionAnalysis.png'}
+      className={`h-[567px] max-h-[567px] w-[1236px] max-w-[1236px]`}
+    />
+    <div
+      className={`mt-[10px] flex h-[328px] w-[293px] flex-col rounded-lg border-[1px] border-[#E3E0E9] p-6`}
+    >
+      <img
+        alt={`Transaction Analysis`}
+        src={'/images/TransactionIllustration.png'}
+        className={`h-[129px] max-h-[129px] w-[224px] max-w-[224px] self-center`}
+      />
+      <div className={`mt-[20px] flex flex-row items-center space-x-2`}>
```

Comment:
In general use Tailwind's values and not arbitrary values, especially since the default ones use rem and not pixels.